### ISSUE TITLE
Landing Page

### ISSUE THREAD

**Issue by iAbhinav** [Dominate-X-Club/Copod#1]
Create home page for [copod](https://copod.space)

This is the place where a Podcast Host will first understand the copod concept. 

Copod is a managed podcast recording studio, situated on West End Marg, New Delhi. Our main focus is hospitality and quality of the podcast recording. 

A host doesn't have to spend lakhs in buying all the mic, camera, lights, mixers and what not. Neither do they need a space, sofa, ambience and an attendant. 

Copod has got them covered!

Before starting work on Homepage, please share inspiration ideas and screenshots here, lets first finalise the theme, design and positioning. And then dive into coding.

Landing page like all other pages should be responsive and shall work perfectly on desktop, mobile and tablets. 

More about copod [here](https://x.com/umunbeing/status/1786106285298983364?s=46)

**Comment by shivam969619** [comment]
Designed a basic landing page should i raise the pr?? or any changes??

**Comment by shivam969619** [comment]
![Screenshot (294)](https://github.com/Dominate-X-Club/Copod/assets/118355740/6959b23e-0779-434f-93b8-d66c8719d40b)

**Comment by shivam969619** [comment]
![Screenshot (295)](https://github.com/Dominate-X-Club/Copod/assets/118355740/2789bbcb-96dd-4259-99d4-1281fed19df2)

**Comment by shivam969619** [comment]
![Screenshot (296)](https://github.com/Dominate-X-Club/Copod/assets/118355740/93d35968-fa63-4553-90cb-9190a3babd0b)

**Comment by code-withAshish** [comment]
> ![Screenshot (294)](https://github.com/Dominate-X-Club/Copod/assets/118355740/6959b23e-0779-434f-93b8-d66c8719d40b)
> 

I think you have replied with this photo 3 times ,could have been a mistake, also the design part of your landing page isn't satisfactory yet the navbar is too wide, content does not have any style, images are placed randomly...
Please work on the design part and submit again.
Thank you

**Comment by shivam969619** [comment]
sure

**Comment by last-developer** [comment]
Didn't find these design much appealing, i think why don't make a minimalist, sleek & modern design

**Comment by iAbhinav** [comment]
> > ![Screenshot (294)](https://github.com/Dominate-X-Club/Copod/assets/118355740/6959b23e-0779-434f-93b8-d66c8719d40b)
> > 
> 
> I think you have replied with this photo 3 times ,could have been a mistake, also the design part of your landing page isn't satisfactory yet the navbar is too wide, content does not have any style, images are placed randomly...
> Please work on the design part and submit again.
> Thank you 

I second that, this UI is very raw. First lets share design inspiration, finalise the design and then go ahead to coding it.

**Comment by shivam969619** [comment]
Any figma file would give an Idea to me ....Will be fine if I share my figma design for landing page??what says??

**Comment by code-withAshish** [comment]
> Any figma file would give an Idea to me ....Will be fine if I share my figma design for landing page??what says??

Do it on the discord channel, we will be having those discussions there only.

**Comment by shivam969619** [comment]
Share discord link

**Comment by code-withAshish** [comment]
> Share discord link

https://discord.com/invite/dPYVyVQy

**Comment by Dey11** [comment]
Any website's UI that you like that we can take inspiration from?

**Comment by lakshya-roonwal** [comment]
how about 
![image](https://github.com/Dominate-X-Club/Copod/assets/76898541/3b314afd-b0ec-4a46-add0-c848390761d7)
Here is URL for the full landing page : https://copod2.vercel.app/

**Comment by shivam969619** [comment]
Looks cool btw

**Comment by iAbhinav** [comment]
> ### How about 
> 
> ![image](https://github.com/Dominate-X-Club/Copod/assets/76898541/3b314afd-b0ec-4a46-add0-c848390761d7)
> 
> Here is URL for the full landing page : https://copod2.vercel.app/

This looks much better, can make it even better. Need some colors over white in some sections

**Comment by lakshya-roonwal** [comment]
Will need a color theme then... who's gonna provide that so that we can stick to the brand's color theme and branding

**Comment by iAbhinav** [comment]
Hey guys, I find these designs good to get inspired from 🤭

What do you think 💭 

A. 

![image](https://github.com/Dominate-X-Club/Copod/assets/6393661/00ddc931-63d5-4c8b-b6c0-a6068cdbaa63)

B. 

![image](https://github.com/Dominate-X-Club/Copod/assets/6393661/2a42ca35-1cf4-4e5c-8be4-2acff093f177)

C. 

![image](https://github.com/Dominate-X-Club/Copod/assets/6393661/8fed5473-5879-42c8-8883-0b05ccea4f52)

**Comment by Iamanshuaditya** [comment]
Hey @iAbhinav made a pull req please look into that

**Comment by iAbhinav** [comment]
We have selected this theme, need help to make the UI sexier

<img width="1440" alt="Screenshot 2024-05-22 at 4 22 54 PM" src="https://github.com/Dominate-X-Club/Copod/assets/6393661/a8cc3166-872b-4a0d-865b-7568720e836f">

**Comment by kalkeshwar** [comment]
@iAbhinav  please assign me . I am ready to work on it.

**Comment by code-withAshish** [comment]
> @iAbhinav  please assign me . I am ready to work on it. 

This issue is not of an assignable nature as the initial part has been made already and now it's on an incremental contribution basis, so feel free to add to this, share your updates on our discord, we will be happy to see what you make and if satisfactory it'll be merged into the main branch.

Thank you
